pkg/security/config/config.go

```diff
@@ -19,30 +20,38 @@ type Policy struct {
 
 // Config holds the configuration for the runtime security agent
 type Config struct {
-	Enabled             bool
-	BPFDir              string
-	PoliciesDir         string
-	EnableKernelFilters bool
-	EnableApprovers     bool
-	EnableDiscarders    bool
-	SocketPath          string
-	SyscallMonitor      bool
-	EventServerBurst    int
-	EventServerRate     int
+	Enabled                            bool
```

Comment:
Would you like to add comments to explain intention behind some of these config settings?

Reply:
Done 👍